docusaurus/docs/adding-typescript.md

```diff
@@ -16,6 +16,7 @@ npx create-react-app my-app --typescript
 
 yarn create react-app my-app --typescript
 ```
+> Note that having `create-react-app` installed globally can cause conflicts when using `--typescript`. If you've previously installed `create-react-app` globally via `npm install -g create-react-app`, we recommend you uninstall the package using `npm uninstall -g create-react-app` to ensure that `npx` always uses the latest version. 
```

Comment:
This first sentence is not accurate. Having Create React App installed globally doesn't create a "conflict" with the `--typescript` flag. It just means you're more likely to have an old version installed that doesn't have the `--typescript` flag. As the last sentence says, you should use `npx` instead of a global install to ensure you're using an up to date version of Create React App.

Reply:
I've made this change but I'm still not sure how to correctly remove the commits from @nagman.